srcpkgs/s6-rc-man-pages/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 's6-rc-man-pages'
+pkgname=s6-rc-man-pages
+version=0.5.4.1.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="mdocml"
```

Comment:
Is the hostmakedepends actually required? The makefiles for this and the other -man-pages packages seem to just install the files.

Reply:
It is not... 
This is probably something I added during testing or on a previous version that was needed and copied it on new templates:(  Removed, thanks!